packages/next/client/image.tsx

```diff
@@ -116,7 +117,9 @@ const {
   path: configPath,
   domains: configDomains,
 } = (process.env.__NEXT_IMAGE_OPTS as any as ImageConfigComplete) ||
+imageConfigRuntime ||
```

Comment:
Can we use a react context provider instead of another variable here?

Reply:
Done, I change to use `useContext()` to update `configDomains`